app/services/preserved_object_handler.rb

```diff
@@ -100,15 +100,16 @@ def update
 
   # expects @incoming_version to be numeric
   # TODO: update existence check timestamps/status per each flavor of comparison?
+  # rubocop:disable Metrics/PerceivedComplexity
```

Comment:
can we indicate this is temporary and what ticket corresponds to its removal?   I hate this noise in the code.

Reply:
alternately, can we just raise the default allowed?